src/PrestaShopBundle/Controller/Admin/Sell/Customer/CustomerController.php

```diff
@@ -838,6 +839,16 @@ private function getErrorMessages(Exception $e)
                     ),
                 ]
             ),
+            BadCustomerRequiredFieldsException::class => $this->trans(
+                'The required field %s is not valid.',
```

Comment:
@LouiseBonnard If someone sets "newsletter partners" as required field and then attempts to create a customer WITHOUT checking the checkbox, I display this message to him.

"The required field optin is not valid."

It's a bit generic, but since this could be used for other required-fields-that-is-not-properly-filled usecases (not only checkboxes) it's the best I can think of.